pkg/list/list_vendor.go

```diff
@@ -0,0 +1,217 @@
+package list
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/lipgloss/table"
+	"github.com/cloudposse/atmos/internal/exec"
+	"github.com/cloudposse/atmos/pkg/config"
+	"github.com/cloudposse/atmos/pkg/schema"
+	"github.com/cloudposse/atmos/pkg/ui/theme"
+	"github.com/cloudposse/atmos/pkg/utils"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	DefaultCSVDelimiter = ","
+	DefaultTSVDelimiter = "\t"
+)
+
+// VendorInfo represents a vendor configuration entry
```

Comment:
Also, the fact we are needing this here is a red flag. We have a robust handling of vendor configs already.